### ISSUE TITLE
drafts information from paginated query List project tasks

### ISSUE THREAD

**Issue by spagliarini** [heartexlabs/label-studio#5983]
Hi, 
I would like 
- to export the annotations from my label studio project and be able to access the 'drafts' information for each task
- to use a paginated query to do so (as 'List project tasks' in the [documentation](https://labelstud.io/api#tag/Projects/operation/api_projects_tasks_list))

I noticed that using the paginated query the information I am looking for is unavailable.

**Describe the solution you'd like**
I'd like to find the 'drafts' information when using a paginated query to export my annotations.

**Describe alternatives you've considered**
I'd like to avoid the usage of [easy export](https://labelstud.io/api#tag/Export/operation/api_projects_export_read) because I don't need to export all the annotations but only the ones from a given page. In addition, it wouldn't be a permanent solution since my amount of data could increase and the solution of exporting everything at the same time wouldn't work anymore. For similar reasons, I would avoid exporting a view since in my situation it would be a dynamic view and not a stable one (I would need to create the view manually any time I want to export my annotations, and this sounds unpractical).

Another option would be to find a way to get the list of task ids without exporting the whole project (i.e., all the tasks with their predictions, annotations, etc.). So far, I haven't found a way to do so through the APIs? Am I missing something?

Thank you so much for your your availability

**Comment by niklub** [comment]
Hi, @spagliarini !

I'd suggest you to try the following method:

1. Install label-studio-sdk with `pip install -U label-studio-sdk`
2. Run the following code:
```python
# Import the SDK and the client module
from label_studio_sdk.client import LabelStudio

LABEL_STUDIO_URL = 'http://..."
API_KEY = '...'
PROJECT_ID = 123

# Connect to the Label Studio API and check the connection
ls = LabelStudio(base_url=LABEL_STUDIO_URL, api_key=API_KEY)

for task in ls.tasks.list(project=PROJECT_ID, fields='all'):
    # get your annotations
    print(task.annotations)
    # get your drafts
    print(task.drafts)
```

The `tasks.list()` method takes care of auto paginated response

To retrieve tasks from a given page, use the following:
```python
for task in ls.tasks.list(..., page=5):
     ...
```

You can get more information about available methods here: https://api.labelstud.io/

**Comment by spagliarini** [comment]
Hi @niklub , 

I tried approaching the problem as you suggested and I realized that an API that can do the job already exists. It's only hidden in the API documentation. 

Here https://labelstud.io/api#tag/Tasks/operation/api_tasks_list, the get tasks list API has no "fields" parameter listed, but it's possible to use it within this API. Moreover, the given example doesn't show "drafts" in the obtained result, while obtaining the drafts is possible. 

Thank you for your suggestions, it was helpful to reach a solution!
